tests/baselines/reference/config/initTSConfig/Default initialized TSConfig/tsconfig.json

```diff
@@ -10,6 +10,31 @@
     // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
+    /* Emit */
```

Comment:
I saw this change when I attempted this too; I don't quite know how this works, but can we hardcode the ordering of these seconds so they don't change? Feels like emit shouldn't appear above the language /lib target settings.

Reply:
(if you're planning on it, no worries; I know this is WIP. I just will totally forget if I don't comment while I remember...)

Reply:
Yep planning on it.. just wanted to get this here so i can easily diff and view, run some tests here instead of locally etc.